server/public/model/config.go

```diff
@@ -1395,7 +1395,7 @@ type ExperimentalAuditSettings struct {
 	FileMaxBackups        *int            `access:"experimental_features,write_restrictable,cloud_restrictable"`
 	FileCompress          *bool           `access:"experimental_features,write_restrictable,cloud_restrictable"`
 	FileMaxQueueSize      *int            `access:"experimental_features,write_restrictable,cloud_restrictable"`
-	AdvancedLoggingJSON   json.RawMessage `access:"experimental_features,write_restrictable,cloud_restrictable"`
```

Comment:
Can you help me understand the effect of this change? This setting would still be set in cloud to our default value, so I am guessing they will see it in their console, but the value will be hardcoded to the env var setting right?

Reply:
cloud_restrictable means that they'd not be able to set it in cloud at all - so if we want to allow them to configure this value, we have to remove it. The intention is that Mattermost would set the value for `MM_EXPERIMENTALAUDITSETTINGS_ADDITIONAL` as we would normally set `AdvancedLoggingJSON`, since that is only allowed to be set via ENV variable. Then, users would be free to adjust AdvancedLoggingJSON however they'd like